project/Dependencies.scala

```diff
@@ -139,7 +139,6 @@ object Dependencies {
     slick,
     slickHikariCP,
     akkaHttp,
-    akkaHttp,
```

Comment:
this was there twice?

Reply:
Yeah... I didn't realize I kept the removal of one in this PR but it certainly doesn't need to be there twice.